lab_5_scrapper/scrapper.py

```diff
@@ -204,12 +350,25 @@ def prepare_environment(base_path: Union[pathlib.Path, str]) -> None:
     Args:
         base_path (Union[pathlib.Path, str]): Path where articles stores
     """
-
+    if not base_path.exists():
+        base_path.mkdir()
+    base_path.rmdir()
```

Comment:
if directory is not empty, it will be problematic

Reply:
**do not resolve comments yourself!**

Reply:
**OK?**

Reply:
Yeah, alright👌